### PR TITLE
1915 weather file2

### PR DESCRIPTION
**Pull request by macumber**

### CODE REVIEW

File: openstudiocore/src/openstudio_lib/OSDocument.cpp [hunk 1]
```diff
@@ -443,6 +445,18 @@ namespace openstudio {
...
+    // convert absolute weather file paths to relative in the model, also copy the epw file to the temp dir
+    // remove the weather file object if something goes wrong
+    bool weatherFileOk = this->fixWeatherFileInTemp(true);
+    if (!weatherFileOk){
+      // weather file in model was reset because the file did not exist or it was not a valid EPW
+      modified = true;
+
+      // connect to slot that would show user error dialog
+      // DLM: Evan todo
+      //QTimer::singleShot(0, this, SLOT(weatherFileReset()));
```
Comment (macumber) [inline openstudiocore/src/openstudio_lib/OSDocument.cpp]: @evanweaver this is where user warning would go, something like:

"Invalid weather file, weather file has been reset"

**Comment by macumber** [comment]
@evanweaver this is ready to go, we need to do all the testing on the builds before we release them

**Comment by evanweaver** [comment]
@macumber agreed, but I don't know how to get in touch with our software test department ;)

**Comment by macumber** [comment]
Don't bother, those guys are a bunch of jerks ;-) .... I just meant going through the usual workflows for testing, I didn't test making an OSM and running in App or PAT

**Comment by evanweaver** [comment]
:)

We are still doing 8.5 first, right?

**Comment by macumber** [comment]
Yeah I am working on it now, doesn't seem to be too many changes

**Pull request merged**